osx/post_installation.sh

```diff
@@ -0,0 +1,113 @@
+PLIST_SRC="$TMP$ORG.plist"
+PLIST_DST"$LAUNCH_AGENTS$ORG.plist"
+
+
+#----------------------------------------------------------------------
+# Functions
+#----------------------------------------------------------------------
+function update_env {
+    # MUST: Make sure we have a KALITE_PYTHON env var that points to Pyrun
+    echo "Updating KALITE_PYTHON environment variable..."
+    launchctl setenv  KALITE_PYTHON "$PYRUN"
+    export KALITE_PYTHON="$PYRUN"
+    if [ $? -ne 0 ]; then
+        echo ".. Abort!  Error/s encountered exporting KALITE_PYTHON '$PYRUN'."
+        exit 1
+    fi
+}
+
+function create_plist {
+    # Create Plist 
+    echo "Now creating '$PLIST_SRC'..."
+    echo "<?xml version='1.0' encoding='UTF-8'?>" >> $PLIST_SRC
+    echo "<!DOCTYPE plist PUBLIC '-//Apple//DTD PLIST 1.0//EN' 'http://www.apple.com/DTDs/PropertyList-1.0.dtd'>" >> $PLIST_SRC
+    echo "<plist version='1.0'>" >> $PLIST_SRC
+    echo "<dict>" >> $PLIST_SRC
+    echo -e "\t<key>Label</key>" >> $PLIST_SRC
+    echo -e "\t<string>org.learningequality.kalite</string>" >> $PLIST_SRC
+    echo -e "\t<key>ProgramArguments</key>" >> $PLIST_SRC
+    echo -e "\t<array>" >> $PLIST_SRC
+    echo -e "\t\t<string>sh</string>" >> $PLIST_SRC
+    echo -e "\t\t<string>-c</string>" >> $PLIST_SRC
+    echo -e "\t\t<string>launchctl setenv KALITE_PYTHON \"$TO_KALITE_PYTHON\"</string>" >> $PLIST_SRC
+    echo -e "\t</array>" >> $PLIST_SRC
+    echo -e "\t<key>RunAtLoad</key>" >> $PLIST_SRC
+    echo -e "\t<true/>" >> $PLIST_SRC
+    echo "</dict>" >> $PLIST_SRC
+    echo "</plist>" >> $PLIST_SRC
+}
+
+#----------------------------------------------------------------------
+# Script
+#----------------------------------------------------------------------
+echo "Now preparing KA-Lite dependencies..."
+
+# Symlink kalite executable to /usr/local/bin
+if [ -f "$KALITE" ]; then
+    echo ".. Found $KALITE executable, it will be removed and will create new one."
+    rm -fr $KALITE
+fi
+$COMMAND_SYMLINK
+
+
+update_env
+
+# Create plist in /tmp and /Library/LaunchAgents folders.
+if [ -f "$PLIST_SRC" ]; then
+    echo ".. Found an existing '$PLIST_SRC', now removing it."
+    rm -fr $PLIST_SRC
+    rm -fr $PLIST_DST
+fi
+create_plist
+sudo cp $PLIST_SRC $PLIST_DST
+
+$PYRUN $SHEBANGCHECK_PATH/shebangcheck.py
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Error encountered running '$SHEBANGCHECK_PATH/shebangcheck.py'."
+    exit 1
+fi
+
+echo "Running manage syncdb..."
+kalite manage syncdb --noinput
+if [ $? -ne 0 ]; then
+    syslog -s -l error "Error encountered running kalite manage syncdb --noinput"
+    exit 1
+fi
+echo "Running manage setup..."
+kalite manage setup --noinput
+if [ $? -ne 0 ]; then
+    syslog -s -l error "Error encountered running kalite manage setup --noinput"
+    # TODO(eduard):  We encountered an error on kalite manage setup --noinput 
+    # REF: https://github.com/learningequality/ka-lite/pull/4630#issuecomment-155562193
+    exit 1
```

Comment:
@djallado You must comment this for now until we have resolved the error when running `kalite manage setup --noinput`.  Or else, this script will never complete.